packages/server/graphql/queries/helpers/suggestedIntegrationHelpers.ts

```diff
@@ -35,34 +36,33 @@ export const useOnlyUserIntegrations = (
 
   // at least 1 person has 1 integration on the team
   const integrationSet = new Set()
-  teamIntegrationsByUserId.forEach(({id}) => integrationSet.add(id))
+  teamIntegrationsByTeamId.forEach(({id}) => integrationSet.add(id))
 
   // the user is integrated against every team integration, use those
   return userIntegrationsForTeam.length === integrationSet.size
     ? userIntegrationsForTeam
     : undefined
 }
 
-export const getTeamIntegrationsByUserId = async (
-  teamId: string
-): Promise<IntegrationByUserId[]> => {
+export const getTeamIntegrationsByTeamId = async (
+  teamId: string,
+  permLookup: PermLookup
```

Comment:
-1 since we're tightly coupling `getPermsByTaskService` to this call, we can just write 
`permLookup: ReturnType<typeof getPermsByTaskService>` and then we have a single source of truth instead of also maintaining a separate interface.